scripts/.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 88
```

Comment:
Why 88 and not 100?

Reply:
Matches `black`.

Reply:
In xbstrap, we configure black to use a line length of 99, we should probably also do that here.

Reply:
ah, I didn't remember doing that, will change this evening